### PR TITLE
dragon-drop: init at git-2014-08-14

### PR DESCRIPTION
**Pull request by jb55**
handy drag and drop source/sink for X

### CODE REVIEW

File: pkgs/tools/X11/dragon-drop/default.nix [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+
+  buildInputs = [ gtk pkgconfig ];
+
+  installPhase = ''
+    mkdir -p $out/bin
+    mv dragon $out/bin
+  '';
+
+  meta = with stdenv.lib; {
+    description = "simple drag-and-drop source/sink for X";
```
Comment (jagajaga) [inline pkgs/tools/X11/dragon-drop/default.nix]: Please start with a capital letter.

**Comment by jb55** [comment]
:+1:

**Comment by pSub** [comment]
Please squash the commits otherwise it looks ready for merge.

**Comment by jb55** [comment]
done

**Pull request merged**